.vscode/settings.json

```diff
@@ -120,6 +120,7 @@
 		"stretchr",
 		"STTY",
 		"stuntest",
+		"subpage",
```

Comment:
maybe "sub-page" instead would avoid angering the spellcheck gods?

Reply:
Would it? There's a lot of props and variables that use `subpage`